src/js/mixins/evented.js

```diff
@@ -366,7 +367,12 @@ function evented(target, options = {}) {
   Obj.assign(target, EventedMixin);
 
   // When any evented object is disposed, it removes all its listeners.
-  target.on('dispose', () => target.off());
+  target.on('dispose', () => {
+    target.off();
+    window.setTimeout(() => {
+      target.eventBusEl_ = null;
```

Comment:
we want to null out this reference so that the reference isn't retained when a component is disposed.